show/main.py

```diff
@@ -1781,6 +1781,119 @@ def policer(policer_name, verbose):
     run_command(cmd, display_cmd=verbose)
 
 
+#
+# 'sflow command ("show sflow ...")
+#
+@cli.group(invoke_without_command=True)
+@click.pass_context
+def sflow(ctx):
+    """Show sFlow related information"""
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    ctx.obj = {'db': config_db}
+    if ctx.invoked_subcommand is None:
+        show_sflow_global(config_db)
+
+#
+# 'sflow command ("show sflow interface ...")
+#
+@sflow.command('interface')
+@click.pass_context
+def sflow_interface(ctx):
+    """Show sFlow interface information"""
+    show_sflow_interface(ctx.obj['db'])
+
+def sflow_appDB_connect():
```

Comment:
What is this function? Looks different from the objective. May be `sflow_sample_rate` ?

Reply:
We need to connect to applDB to get default sample-rate per port if the user doesn't configure them manually.